addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -45,7 +45,7 @@ export class WebsiteLoader extends Component {
                         if (this.featuresInstallInfo.nbInstalled === this.featuresInstallInfo.total) {
                             clearInterval(trackModulesInterval);
                         }
-                    }, 2000);
+                    }, 300);
```

Comment:
2000 was a bit high imo, 300 is too small and was just for the test/wip.
I would expect the configurator to take like ~4-10 sec max if you select 2-4 apps on the saas /trial, could you have a look?
But maybe 2000 was fine, if the configurator is fast, the user probably won't leave anyway, and if it's slow, 2 sec for an update is okay. I guess you discussed that with @sobo-odoo and 2 sec was judged fine anyway? You can probably ignore this :+1:

Reply:
2000 was more of a default suggestion, I didn't discuss it, but yes what you wrote was my train of thought (it's useful when it's slow, and if it's slow 2s is an acceptable interval to show there is some progress). I've updated it to 1000, it might be a good balance. What do you think?

Btw, I tested the configurator on trial, and it's taking at least as long as a local server or the runbot.

Reply:
I don't have a strong opinion, it's a balance between not taking too many ressources and having a smooth progress.
I think anything between 1s and 2s might be correct.
The more I think about it the more I think it was fine (2s) :+1:
I'll let @sobo-odoo decide anyway